.circleci/config.yml

```diff
@@ -573,8 +573,6 @@ jobs:
       - run:
           command: SEGMENT_API_KEY=$SEGMENT_API_KEY_STAGING SENTRY_AUTH_TOKEN=$SENTRY_AUTH_TOKEN SENTRY_DSN=$SENTRY_DSN SENTRY_ENV=$SENTRY_ENV_STAGING POSTHOG_API_KEY=$POSTHOG_API_KEY_STAGING npm run build
           no_output_timeout: "30m"
-      - run:
-          command: rm build/*.map
```

Comment:
Could we leave this in? I guess it's no harm.

Just if we re-enable sourcemaps, we shouldn't forget to avoid them from being uploaded to production.

Reply:
We do need to remove these unfortunately - otherwise the pipeline fails with `rm: cannot remove 'build/*.map': No such file or directory`